pokemongo_bot/cell_workers/pokemon_optimizer.py

```diff
@@ -313,7 +313,7 @@ def get_best_pokemon_for_rule(self, pokemon_list, rule):
         sorted_pokemon = self.sort_pokemon_list_to_keep(pokemon_list, rule)
 
         if len(sorted_pokemon) == 0:
-            return ([], [], [])
+            return ([], [], [], [])
```

Comment:
Since buddy was added this function must return tuple with length of 4, not 3 as it was before.